.github/workflows/c.yml

```diff
@@ -59,3 +59,16 @@ jobs:
         run: ./build/ml_kem_bench
         if: ${{ matrix.os != 'windows-latest' && (github.event_name == 'merge_group' || github.event_name == 'workflow_dispatch') }}
 
+  mq_status:
```

Comment:
Can you move this out to the actions repo to make it re-usable and avoid duplicating the code?